cmake/FindEs2kDriver.cmake

```diff
@@ -0,0 +1,151 @@
+# FindEs2kDriver.cmake - import ES2K P4 Driver (SDE).
+#
+# Copyright 2022-2023 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+#
+
+# This module requires that SDE_INSTALL_DIR be defined.
+# It needs additional work to make it a proper CMake find-module.
+
+#-----------------------------------------------------------------------
+# Define SDE_INCLUDE_DIR
+#-----------------------------------------------------------------------
+find_path(SDE_INCLUDE_DIR
+    NAMES "bf_switchd/bf_switchd.h"
+    PATHS ${SDE_INSTALL_DIR}/include
+)
+if(NOT SDE_INCLUDE_DIR)
+  message(FATAL_ERROR "bf_switchd.h not found")
```

Comment:
Why are we looking specifically for the bf_switchd.h? Or is it just one file that was picked as that was common across all targets?

Reply:
Just looking for something to verify that SDE_INSTALL_DIR points to the root of an SDE install tree and that seemed unlikely to change.